### ISSUE TITLE
Crash when accidently calling to deactivate license with no license in place

### ISSUE THREAD

**Issue by charlieMonroe** [PaddleHQ/Mac-Framework#161]
If there is no license in place and `deactivateLicense()` is called on `Paddle`, it should be gracefully handled, but instead, the app crashes with the following message:

```
 *** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]
```

**Comment by GregularExpressions** [comment]
Hey @charlieMonroe thanks for reporting this, I agree with you `deactivateLicense()` should simply fail silently on production and maybe output a console warning in dev.

I'll add it to the backlog.

**Comment by lvandal** [comment]
This is most likely the same bug as issue #159

**Comment by GregularExpressions** [comment]
This should be fixed in [v3.0.26](https://github.com/PaddleHQ/Mac-Framework/releases/tag/v3.0.26)
